string name
int32 transporter_num
int32 start
int32 finish
int32 duration

int32 PICKUP = 0
int32 DELIVERY = 1
int32 task_type

int32 from_machine_num
int32 to_machine_num
